packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -2224,6 +2224,27 @@ QUnit.module('keyboard navigation', {
         assert.notOk($items.eq(0).hasClass(DX_STATE_FOCUSED_CLASS), 'first item lose focus');
     });
 
+    [
+        [{ text: 'item1', items: [{ name: 'item_1_1' }] }],
+        [{ icon: 'imageCssClass', items: [{ name: 'item_1_1' }] }],
+        [{ text: 'item1', icon: 'imageCssClass', items: [{ name: 'item_1_1' }] }],
+    ].forEach(item => {
+        checkStyleHelper.testInChromeOnDesktopActiveWindow('root item text should not use color property from item template when focused (T1227670)', function(assert) {
+            this.instance.option('items', item);
+
+            $(this.instance.itemsContainer())
+                .find('.' + DX_MENU_ITEM_CLASS)
+                .eq(0)
+                .trigger('dxhoverstart')
+                .trigger('dxclick');
+
+            const $rootMenuItem = $(this.instance.itemElements().eq(0));
+
+            assert.ok($rootMenuItem.hasClass(DX_MENU_ITEM_EXPANDED_CLASS), 'root item should have expanded class');
+            assert.strictEqual(checkStyleHelper.getColor($rootMenuItem[0]), 'rgb(51, 51, 51)', 'color');
```

Comment:
this test is evergreen now :(

Reply:
That's why I separated it state-focused because when I put the solution on menu-item-expanded, it always evergreen :( I'll investigate this.

Reply:
That's because root menu item is not focused after popup opening.
Please, call `instance.focus()` and also add an assert that focused class is applied.

Reply:
About that, I actually tried this method last week, after triggering to expand, I can't seem to focus it even with instance.focus(). Also tried $rootItem.focus() but doesn't work.

Reply:
checked - yeah, i see that there is smth strange with `focus` method in native js. You can do it this way: `this.$element.trigger('focusin');`